src/Api/AdminConsole/Models/Response/Organizations/OrganizationUserResponseModel.cs

```diff
@@ -85,9 +87,12 @@ public class OrganizationUserDetailsResponseModel : OrganizationUserResponseMode
 
 public class OrganizationUserUserDetailsResponseModel : OrganizationUserResponseModel
 {
+    // Since the OrganizationUserUserDetailsQueryResponse object already has the permissions
+    // converted from json. Add it here as an optional param. If included the base method will
+    // use the permissions if not it will conver
     public OrganizationUserUserDetailsResponseModel(OrganizationUserUserDetails organizationUser,
-        bool twoFactorEnabled, string obj = "organizationUserUserDetails")
-        : base(organizationUser, obj)
+        bool twoFactorEnabled, string obj = "organizationUserUserDetails", Permissions permissions = null)
+        : base(organizationUser, obj, permissions: permissions)
```

Comment:
Again, we avoid all of this if we omit it from the query and let consumers decide if/when they want to deserialize this.